specification.markdown

```diff
@@ -737,7 +735,7 @@ This document contains definitions of all EBML Elements of the EBML Header.
 
 name: EBML
 
-path: `1*1(\EBML)`
+path: `\EBML`
```

Comment:
I don't understand this change and the subsequent ones. If the PathMinOccurrence is not present then it is understood to be 0, so this edit seems to mean that the EBML Header is no longer required.

Reply:
`PathMinOccurrence` and `PathMaxOccurrence` are only use for global elements which `\EBML` and the following ones are not:
> PathMinOccurrence represents the minimum number of element path required between the EBMLFixedParent and the Global Element EBMLElementPath.
> PathMaxOccurrence represents the maximum number of element path possible between the EBMLFixedParent and the Global Element EBMLElementPath

I have to rework the global elements path to make this clearer (#326)

Basically, most pathes don't have occurences in them anymore. Only the Global Elements do (for now).